mmv1/products/databasemigrationservice/connectionprofile.yaml

```diff
@@ -0,0 +1,496 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ConnectionProfile'
+self_link: 'projects/{{project}}/locations/{{location}}/connectionProfiles/{{connection_profile_id}}'
+base_url: 'projects/{{project}}/locations/{{location}}/connectionProfiles'
+create_url: 'projects/{{project}}/locations/{{location}}/connectionProfiles?connectionProfileId={{connection_profile_id}}'
+update_verb: :PATCH 
+update_mask: true
+description: 'A connection profile definition.'
+references: !ruby/object:Api::Resource::ReferenceLinks
```

Comment:
your example tests are not getting generated/documented. You'll need an `examples` block referencing them in this file.